hugo/content/3_how-tos/33_software-development/334_deploy-packages.md

```diff
@@ -35,16 +35,27 @@ To deploy a package in rapyuta.io, follow the steps:
 
     | Field | Description |
     | ---   | --- |  
-    | Name | Specify the deployment name. |
+    | Name of deployment | Specify the deployment name. |
     | Labels | A set of key value pairs used to filter out resources. To add a label, click **Add label**.|
-    |Refresh the list of online devices | Click to retrieve an updated list of online devices.|
-    | Component Details | The component of the package can have either device or cloud runtime. If the runtime component is device, select the device from the **Select device for deploying the component** dropdown list. |
 
-{{%notice info%}}    
-The list of devices is a set of online devices, which are pre-filtered to match the architecture (amd64, arm32v7, arm64v8) and device runtime (docker or preinstalled) required by the component in question.
-{{% /notice %}}
-
-4. Adding Volumes <br>
+4. Adding Configurations <br>
+   Auto sync the configuration parameters present on rapyuta.io with their corresponding configs in the cloud deployments. 
+   {{%notice info%}}    
+The deployment labels are used for filtering the correct configuration files inside the configuration tree.
```

Comment:
Along with this, pls also add another info with following details,

The executables for which the syncing has to be enabled has to have either 'wget' or 'curl' baked in the image/build.